### PR TITLE
Remove OONIBrowser and its related classes

### PR DESCRIPTION
**Pull request by lorenzoPrimi**

### CODE REVIEW

File: ooniprobe/AppDelegate.mm [hunk 1]
```diff
@@ -79,54 +78,17 @@ -(void)handleNotification:(NSDictionary*)userInfo :(UIApplication *)application{
...
-            links = [[NSMutableArray alloc] init];
-            [links addObject:[[userInfo objectForKey:@"payload"] objectForKey:@"href"]];
-            if ([[userInfo objectForKey:@"payload"] objectForKey:@"alt_hrefs"]){
-                NSArray *alt_href = [[userInfo objectForKey:@"payload"] objectForKey:@"alt_hrefs"];
-                [links addObjectsFromArray:alt_href];
-            }
             [MessageUtility alertWithTitle:nil
                                    message:[NSString stringWithFormat:@"%@",[[userInfo objectForKey:@"aps"] objectForKey:@"alert"]]
-                                  okButton:okButton
                                     inView:self.window.rootViewController];
```
Comment (bassosimone) [inline ooniprobe/AppDelegate.mm:83]: what is this alert doing?

File: ooniprobe/AppDelegate.mm [hunk 2]
```diff
@@ -79,54 +78,17 @@ -(void)handleNotification:(NSDictionary*)userInfo :(UIApplication *)application{
     UIApplicationState state = [application applicationState];
     if (state == UIApplicationStateActive)
     {
-        if ([type isEqualToString:@"open_href"]){
```
Comment (bassosimone) [inline ooniprobe/AppDelegate.mm:82]: can we remove the whole method? what are we doing in it now?

File: ooniprobe/AppDelegate.mm [hunk 3]
```diff
@@ -79,54 +78,17 @@ -(void)handleNotification:(NSDictionary*)userInfo :(UIApplication *)application{
...
--(void)openBrowser __deprecated {
-    dispatch_async(dispatch_get_main_queue(), ^{
-        UIStoryboard *mainStoryboard = [UIStoryboard storyboardWithName:@"OONIRun" bundle: nil];
-        UINavigationController *nvc = [mainStoryboard instantiateViewControllerWithIdentifier:@"browserNC"];
-        BrowserViewController *bvc = (BrowserViewController*)[nvc.viewControllers objectAtIndex:0];
-        [bvc setUrlList:links];
-        [self.window.rootViewController presentViewController:nvc animated:YES completion:nil];
-    });
-}
+
```
Comment (bassosimone) [inline ooniprobe/AppDelegate.mm:91]: no need to add this extra blank line

File: ooniprobe/AppDelegate.mm (hunk 1 above)
Comment (lorenzoPrimi) [reply ooniprobe/AppDelegate.mm:83]: Display the notification if the app is running

File: ooniprobe/AppDelegate.mm (hunk 2 above)
Comment (lorenzoPrimi) [reply ooniprobe/AppDelegate.mm:82]: The whole method is needed to handle notification

File: ooniprobe/AppDelegate.mm (hunk 3 above)
Comment (lorenzoPrimi) [reply ooniprobe/AppDelegate.mm:91]: done

**Pull request merged**